test/ng/documentSpec.js

```diff
@@ -27,3 +27,29 @@ describe('$document', function() {
     });
   });
 });
+
+
+describe('$$isDocumentHidden', function() {
+  it('should listen on the visibilitychange event', function() {
+    var doc;
+
+    var spy = spyOn(document, 'addEventListener').and.callThrough();
+
+    inject(function($$isDocumentHidden, $document) {
+      expect(spy.calls.mostRecent().args[0]).toBe('visibilitychange');
+      expect(spy.calls.mostRecent().args[1]).toEqual(jasmine.any(Function));
```

Comment:
I used that before, but there's a difference between jquery 2.1 and jqlite, jquery calls addEventListener with the third argument set to false, jqlite doesn't set the argument. That's why I used mostRecent.